images/cassandra-reaper/tests/smoke-test.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+function cleanup() {
+  helm uninstall k8ssandra-operator -n k8ssandra-operator-${RANDOM}
+  helm uninstall cert-manager -n cert-manager-${RANDOM}
+}
+
+trap cleanup EXIT
+
+# list all the storageclasses
+
+kubectl get storageclass
+
+# we have to install cert-manager first
+
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager-${RANDOM} jetstack/cert-manager \
+     --namespace cert-manager-${RANDOM} --create-namespace --set installCRDs=true
```

Comment:
Each time you get `${RANDOM}` it'll give you a different value, so this won't work.

You should do something like

```
name=cert-manager-${RANDOM}
namespace=cert-manager-${RANDOM}

helm install $name jetstack/cert-manager \
     --namespace $namespace --create-namespace --set installCRDs=true

kubectl wait --for=condition=ready pod --selector app.kubernetes.io/instance=cert-manager --namespace $namespace
...
```

Reply:
why? I created an environment variable called $RANDOM in the tests through a terraform resource random_pet ? 🤔

Reply:
done